src/javascript/app/pages/deriv_iframe.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { isOAuth2Enabled } from '../../_common/auth';
+
+const DerivIFrame = () => (
+    <iframe
+        id='localstorage-sync'
+        style={{ display: 'none', visibility: 'hidden' }}
+        sandbox='allow-same-origin allow-scripts'
+    />
+);
+
+export const init = () => {
+    const isAuthEnabled = isOAuth2Enabled();
+
+    if (!isAuthEnabled) ReactDOM.render(<DerivIFrame />, document.getElementById('deriv_iframe'));
+};
```

Comment:
I moved the `DerivIframe` component to the `javascript` folder so that it can be conditionally rendered to be included/excluded based on the feature flag returned by `isOauth2Enabled`.
If the Hydra authentication is enabled, we will not be using the Deriv iframe to sync login states anymore

Reply:
so we want to disable syncing in certain scenarios?

Reply:
We only disable it when the Hydra authentication feature flag is enabled, which we turn on/off from Growthbook

Reply:
Because one of our aim for this feature is to remove the use of Deriv iframe sync entirely and migrate these syncing logic into the Hydra service, which is on BE's side